LottieSample/src/main/kotlin/com/airbnb/lottie/samples/ShowcaseViewModel.kt

```diff
@@ -24,6 +24,7 @@ class ShowcaseViewModel(application: Application) : AndroidViewModel(application
         disposables.add(lottiefilesService.getCollection(COLLECTION)
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
+                .retry(3)
```

Comment:
some country visit [https://www.lottiefiles.com](https://www.lottiefiles.com) spent a lot of time, and often occur SocketTimeoutException, so I want retry 3 times when occur exception.